dash/dash.py

```diff
@@ -875,6 +910,10 @@ def index(self, *args, **kwargs):  # pylint: disable=unused-argument
 
         # use self.title instead of app.config.title for backwards compatibility
         title = self.title
+        pages_metas = ""
+        if self.use_pages:
+            pages_metas = self._pages_meta_tags()
+            title = ""
```

Comment:
`self._pages_meta_tags` could return a tuple  `pages_metas, title = self._pages_meta_tags()`

Reply:
fixed - no need for tuple